src/pytest_forked/__init__.py

```diff
@@ -88,4 +90,24 @@ def report_process_crash(item, result):
         rep.sections.append(("captured stdout", result.out))
     if result.err:
         rep.sections.append(("captured stderr", result.err))
+
+    xfail_marker = item.get_closest_marker('xfail')
+    if not xfail_marker:
+        return rep
+
+    rep.outcome = "skipped"
+    rep.wasxfail = (
+        "reason: {xfail_reason}; "
+        "pytest-forked reason: {crash_info}".
+        format(
+            xfail_reason=xfail_marker.kwargs['reason'],
+            crash_info=info,
+        )
+    )
+    warnings.warn(
+        'pytest-xdist xfail support is imperfect at the moment and may '
```

Comment:
I would proposal the word incomplete, also wrong plugin names

Reply:
Oh, missed that... thanks!

Reply:
fixed